test/e2e/helpers.js

```diff
@@ -103,6 +114,9 @@ async function withFixtures(options, testSuite) {
   } finally {
     await fixtureServer.stop();
     await ganacheServer.quit();
+    if (ganacheOptions?.concurrent) {
+      secondaryGanacheServer.quit();
```

Comment:
There is an `await` missing here

```suggestion
      await secondaryGanacheServer.quit();
```

Reply:
Good?